specs/vm/instruction_set.md

```diff
@@ -1417,6 +1418,22 @@ This modifies the `balanceRoot` field of the appropriate output.
 `messageID` is added to the `OutputMessage` Merkle tree as part of block header.
 TODO: document output messages merkle tree construction and maintenance and link here
 
+### SCWQ: State clear sequential 32 byte slots
+
+|             |                                                                               |
+|-------------|-------------------------------------------------------------------------------|
+| Description | A sequential series of 32 bytes is cleared from the current contract's state. |
+| Operation   | ```STATE[MEM[$rA, 32 * rB]] = None;```                                        |
```

Comment:
I'm assuming the intention is to provide a starting key and delete the next `n` entries, rather than providing an array of keys to bulk delete from the db?

```suggestion
| Operation   | ```STATE[MEM[$rA, 32], 32  * $rB] = None;```                                        |
```